package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
I think you should not allow importing outside of main and module entry points, but that is really up to you.  What are the benefits of getting from src?  I think the costs outway the benefits on this  especially as more transforms are added.  I think we should stick to the webpack recommendation.

> Here's a use-case. The library gets included on https://cdnjs.com/. They use npm to get the latest version, and they add dist/*.js to the CDN.

when you npm publish you will publish the entire dist/ folder so they will have the minified and non minified bundles.

>Here's another: I don't use webpack, but I do use npm to manage my dependencies. So I create an express route to point to dist/cross-domain-utils.min.js

Wouldnt they just put a public static folder like
```
app.use(express.static('./node_modules/cross-domain-utils/dist'))
```
I don't know if you need to use the files array for this or not.  I really didn't think of this use case.  What about doing main: "dist/minified" and then files: [dist/unminified]?  By default they will get minified and can specify unminified if they want...  Not sure we even need to do that since there are sourcemaps...  up to you.

> I haven't yet heard a good answer for how this is going to work for:

What is the point of flow runtime?  the IDE kinda takes care of runtime type checking.

When i chunked out the IE support I am only importing it if the config.IE is set to true, which it is by default.  For script includes i would say we could add post-robot-no-ie to the script tag to not import the IE support.  For npm this will be trickier.  I haven't worked with the innards of post-robot in awhile but I would maybe require the user to instantiate via constructor that accepts a config.  

```
pr = new PostRobot(config)
```
script include could then auto instantiate having defaults and merging window.postRobotConfig and maybe the script attributes.

> How will dist/module be included in the npm package?
Not sure what you mean here.  I am assuming npm will packate dist/module since it is defined by the module property.  Guess you will have to check when you publish the module since I can't publish.